stripe.go

```diff
@@ -315,7 +330,24 @@ func (s *BackendConfiguration) Do(req *http.Request, v interface{}) error {
 
 	start := time.Now()
 
-	res, err := s.HTTPClient.Do(req)
+	var res *http.Response
+	var err error
+	for retry := 0; ; {
+		res, err = s.HTTPClient.Do(req)
+		if s.shouldRetry(err, res, retry) {
+			sleep := s.sleepTime(retry)
+			if LogLevel > 2 {
```

Comment:
I'm kind of thinking that maybe this should be `LogLevel > 0`. Errors that occur here are still errors and should probably appear in the logs.

Reply:
ok